apps/system/test/unit/nfc_handover_manager_test.js

```diff
@@ -295,163 +594,200 @@ suite('Nfc Handover Manager Functions', function() {
 
       nfcHandoverManager.sendFileQueue = [];
       nfcHandoverManager.start();
-      invokeBluetoothGetDefaultAdapter();
     });
 
     teardown(function() {
-      MockBluetooth.enabled = false;
-      MockService.mIsFileTransferInProgress = false;
-
-      spySendNDEF.restore();
       nfcHandoverManager.sendFileQueue = [];
       nfcHandoverManager.stop();
     });
 
     test('"nfc-manager-send-file" results in handover request sent to peer',
```

Comment:
More up to date name needed. Perhaps 'Proceed with file transfer if no concurrent filer transfer is in progress.'